Thirdweb/Thirdweb.Transactions/ThirdwebTransaction.cs

```diff
@@ -357,7 +357,33 @@ public static async Task<string> Send(ThirdwebTransaction transaction)
 
         var rpc = ThirdwebRPC.GetRpcInstance(transaction._wallet.Client, transaction.Input.ChainId.Value);
         string hash;
-        if (await Utils.IsZkSync(transaction._wallet.Client, transaction.Input.ChainId.Value).ConfigureAwait(false) && transaction.Input.ZkSync.HasValue)
+
+        if (transaction.Input.AuthorizationList != null)
+        {
+            var authorization = transaction.Input.AuthorizationList[0];
+            hash = await rpc.SendRequestAsync<string>(
+                    "wallet_sendTransaction",
```

Comment:
what wallet supports this rpc method currently? 

also shouldnt that just be handled in side IThirdwebWallet.SendTransaction ?